FP-functions/functional/functional.js

```diff
@@ -0,0 +1,533 @@
+     * @returns {HTMLInputElement} - Input element
+     */
+    function Input({id, text, appclass, type, onChange, onInput, onSearch, placeHolder}) {
+        const input = document.createElement("input");
+        input.id = id;
+        input.onchange = onChange;
+        input.oninput = onInput;
+        input.onsearch = onSearch;
+        input.type = type;
+        input.placeholder = placeHolder;
+        input.classList = appclass;
+        return input;
+    }
+
+    /**
+     * Div component
+     * @param id {string}
+     * @param text {string}
+     * @param appclass {string}
+     * @returns {HTMLDivElement} - Div element
+     */
+    function Div({id, text, appclass}) {
+        const div = document.createElement("div");
+        div.id = id;
+        div.innerHTML = text;
+        div.classList = appclass;
+        return div;
+    }
+
+
+    /**
+     * App container
+     * @returns {HTMLDivElement} - The app container
+     */
+    function App() {
+        let initTasks = [
+            "Task 1 Title", 
+            "Task 2 Title", 
+            "Task 3 Title"
+            ];
+        let initComplete = [
+            "Completed Task 1 Title", 
+            "Completed Task 2 Title"
+            ];
+        
+        localStorage.getItem("initTasks") ? initTasks = localStorage.getItem("initTasks").split(";"): '';
+        localStorage.getItem("initComplete") ? initComplete = localStorage.getItem("initComplete").split(";"): '';
+        
+        const [items, setItems] = useState(initTasks);
+        const [itemsComplete, setItemsComplete] = useStateComplete(initComplete);
+
+        function addCompleteItem(item) {
+            setItemsComplete([...itemsComplete, item]);
+        }
+
+        function ItemComplete(element) {      
+            const newitem = removeItem(element);
+            console.log(newitem);
+            setItemsComplete([...itemsComplete, newitem]);
+        }
+
+        function ItemUnComplete(element) {
+            const newitem = removeCompletedItem(element);
+            console.log(newitem);
+            setItems([...items, newitem]);
+        }
+
+        function removeItem(element) {
+            const parent = element.srcElement.parentElement.id;
+            const label = parent.replace("task_", "taskLabel_");
+            const item = document.getElementById(label);
+            const removed = items.splice(items.indexOf(item.innerHTML), 1);
+            
+            setItems(items);
+            return removed[0];
+        }
+
+        function removeCompletedItem(element) {
+            const parent = element.srcElement.parentElement.id;
+            const label = parent.replace("complete_", "completeLabel_");
+            const item = document.getElementById(label);
+            const removed = itemsComplete.splice(itemsComplete.indexOf(item.innerHTML), 1);
+
+            setItemsComplete(itemsComplete);
+            return removed[0];
+        }
+
+        function AddRows() {
```

Comment:
You have created component for dev/input and etc, lets try create (in another pr) component for completed/uncompleted rows